sdk/qumulo/Azure.ResourceManager.Qumulo/api/Azure.ResourceManager.Qumulo.net8.0.cs

```diff
@@ -157,21 +163,52 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Qumulo.Models.QumuloFileSystemResourceProperties>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct QumuloProvisioningState : System.IEquatable<Azure.ResourceManager.Qumulo.Models.QumuloProvisioningState>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public QumuloProvisioningState(string value) { throw null; }
+        public static Azure.ResourceManager.Qumulo.Models.QumuloProvisioningState Accepted { get { throw null; } }
+        public static Azure.ResourceManager.Qumulo.Models.QumuloProvisioningState Canceled { get { throw null; } }
+        public static Azure.ResourceManager.Qumulo.Models.QumuloProvisioningState Creating { get { throw null; } }
+        public static Azure.ResourceManager.Qumulo.Models.QumuloProvisioningState Deleted { get { throw null; } }
+        public static Azure.ResourceManager.Qumulo.Models.QumuloProvisioningState Deleting { get { throw null; } }
+        public static Azure.ResourceManager.Qumulo.Models.QumuloProvisioningState Failed { get { throw null; } }
+        public static Azure.ResourceManager.Qumulo.Models.QumuloProvisioningState Succeeded { get { throw null; } }
+        public static Azure.ResourceManager.Qumulo.Models.QumuloProvisioningState Updating { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.Qumulo.Models.QumuloProvisioningState other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Qumulo.Models.QumuloProvisioningState left, Azure.ResourceManager.Qumulo.Models.QumuloProvisioningState right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Qumulo.Models.QumuloProvisioningState (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Qumulo.Models.QumuloProvisioningState left, Azure.ResourceManager.Qumulo.Models.QumuloProvisioningState right) { throw null; }
+        public override string ToString() { throw null; }
     }
     public partial class QumuloUserDetails : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Qumulo.Models.QumuloUserDetails>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Qumulo.Models.QumuloUserDetails>
     {
-        public QumuloUserDetails() { }
+        public QumuloUserDetails(string email) { }
```

Comment:
we are changing a property to required, this is definitely a breaking change.
Would the old api-version just throw error when this property is not specified?